### PR TITLE
menu added

### PR DESCRIPTION
**Pull request by codysmith287**

### CODE REVIEW

**Review by vandickersonjr — CHANGES_REQUESTED** [review]
The menu looks good. 
Once you add the Resturante heading for the address, than you will be good to merge. 

123 Fulton Street							
Fresno, CA 93721
P: (123)456-7890

File: css/styles.css [hunk 1]
```diff
@@ -89,3 +95,93 @@ max-height: 250px;
...
+/* QUOTE */
+
+#review {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  min-height: 600px;
+  background-image: url('../img/food_review.jpg');
+  background-size: cover;
+  background-repeat: no-repeat;
```
Comment (melodyhawkins) [inline css/styles.css:108]: I don't think that you need this. Background-size:cover should prevent the need for no-repeat.

File: css/styles.css [hunk 2]
```diff
@@ -89,3 +95,93 @@ max-height: 250px;
...
+  background-attachment: fixed;
+}
+
+#review p {
+  color: #FFFFFF;
+  font-family: 'Raleway', san-serif;
```
Comment (melodyhawkins) [inline css/styles.css:114]: sans-serif

File: css/styles.css [hunk 3]
```diff
@@ -89,3 +95,93 @@ max-height: 250px;
...
+  font-size: 2em;
+  text-shadow: 1px 1px 5px #0A0A0A;
+}
+
+#review .container {
+  background-color: (20, 20, 20, .6);
```
Comment (melodyhawkins) [inline css/styles.css:120]: rgba(....)

File: css/styles.css [hunk 4]
```diff
@@ -89,3 +95,93 @@ max-height: 250px;
...
+
+#contact form {
+  background-color: #FF6633;
+  border: 1px solid #FFFFFF;
+  padding: 15px;
+}
+
+#contact .form-group span {
+  color: #FF0D06;
+  text-shadow: 1px 1px 5px #FFFFFF;
```
Comment (melodyhawkins) [inline css/styles.css:175]: I don't remember adding the text-shadow in class but if you have it, we probably did. :-)

File: index.html [hunk 5]
```diff
@@ -102,9 +102,213 @@ <h2>1989</h2>
...
+	<!-- END QUOTE -->
+
+<div class="container-fluid" id="menu">
+	<div class="container">
+		<div class="row text-center">
+			<div class="col-md-12">
+				<h1>Menu</h1>
+			</div>
+		</div>
+		<div class="row">
```
Comment (melodyhawkins) [inline index.html:125]: Mine reads: div class="row menu-tabs" Not sure if that was from class or something I added.

File: index.html [hunk 6]
```diff
@@ -102,9 +102,213 @@ <h2>1989</h2>
...
+								<p class="text-justify">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Deserunt perspiciatis veritatis sunt, obcaecati aut omnis, laborum provident amet possimus dicta!</p>
+			    		</div>
+			    		<div class="col-md-3">
+								<img src="img/bruschetta.jpg" alt="" class="img-responsive img-thumbnail">
+								<h3>Appetizer 4</h3>
+								<p class="text-justify">Lorem ipsum dolor sit amet, consectetur adipisicing elit. Deserunt perspiciatis veritatis sunt, obcaecati aut omnis, laborum provident amet possimus dicta!</p>
+			    		</div>
+			    	</div>
+			    </div>
+					<div role="tabpanel" class="tab-pane fade" id="lunch">
```
Comment (melodyhawkins) [inline index.html:161]: Is fade something she told us about adding in class?

**Review by melodyhawkins — APPROVED** [review]
Good job! Just a few comments. If you have any questions let me know.

**Review by vandickersonjr — APPROVED** [review]

File: css/styles.css (hunk 2 above)
Comment (codysmith287) [reply css/styles.css:114]: Good eye, thanks.

File: index.html (hunk 6 above)
Comment (codysmith287) [reply index.html:161]: It was just a side mention when I asked Rocky about the effect.

**Pull request merged**